evento/src/views/MainPage/index.js

```diff
@@ -14,15 +14,23 @@ const tabs = [
 ];
 
 class MainPage extends Component {
+	constructor() {
+		super();
+		this.state = { filter: '' };
+	}
+
+	updateFilter(evt) {
+		this.setState({ filter: evt.target.value });
+	}
 
 	render() {
 		return (
 			<div className="MainPage">
-				<SearchBar />
+				<SearchBar updateFilter={this.updateFilter.bind(this)}/>
```

Comment:
.bind(this) is pretty ugly :/ One option would be to use arrow function here as well (as it automatically binds to 'this'), like (filterValue) => this.updateFilter(filterValue)